pkgs/development/python-modules/ubelt/default.nix

```diff
@@ -51,11 +51,16 @@ buildPythonPackage rec {
     export HOME=$TMPDIR
   '';
 
-  disabledTests = lib.optionals stdenv.hostPlatform.isDarwin [
-    # fail due to sandbox environment
-    "CacheStamp.expired"
-    "userhome"
-  ];
+  disabledTests =
+    [
+      # fails on python 3.13, see https://github.com/Erotemic/ubelt/issues/157
+      "import_module_from_name"
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      # fail due to sandbox environment
+      "CacheStamp.expired"
+      "userhome"
```

Comment:
Perhaps this test would work by using writableaTmpDirAsHomeHook ?

Reply:
I tried this and it works locally, but it still seems to be failing on borg: https://github.com/NixOS/nixpkgs/pull/401838/checks?check_run_id=43429648025

I'll revert.